CHANGELOG.adoc

```diff
@@ -46,6 +46,7 @@ The changes are:
 ** with new derived reference `terminatedOccurrenceArgument : Expression`
 - https://github.com/eclipse-syson/syson/issues/946[#946] [libraries] All standard libraries have been updated to comply with the SysML Beta 2.3 specification.
 - https://github.com/eclipse-syson/syson/issues/982[#982] [metamodel] `Membership#isDistinguishableFrom` derived attribute has been implemented.
+- https://github.com/eclipse-syson/syson/issues/992[#992] [Export] Implement SysML export of ConcernDefinition, ConcernUsage and StakeholderMembership
```

Comment:
Could you with surround SysMLv2 concepts with '`' please?

Reply:
[Export] => [export]

Reply:
Done.